Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -616,6 +616,9 @@ void NetworkProcess::destroySession(PAL::SessionID sessionID, CompletionHandler<
 #endif
 
     if (auto session = m_networkSessions.take(sessionID)) {
+        auto dataStoreIdentifier = session->dataStoreIdentifier();
+        UNUSED_PARAM(dataStoreIdentifier);
+        RELEASE_LOG(Storage, "%p - NetworkProcess::destroySession sessionID=%" PRIu64 " identifier=%s", this, sessionID.toUInt64(), dataStoreIdentifier ? dataStoreIdentifier->toString().utf8().data() : "null"_s);
```

Comment:
`%" PUBLIC_LOG_STRING "` assuming there is nothing sensitive in the databaseStoreIdentifier.